app/src/main/kotlin/com/hedvig/app/feature/insurance/ui/detail/ContractExt.kt

```diff
@@ -45,6 +44,7 @@ fun InsuranceQuery.Contract.toMemberDetailsViewState() =
       null
     },
     change = YourInfoModel.Change,
+    cancelInsurance = YourInfoModel.CancelInsuranceButton(id), // todo ensure this is the correct ID
```

Comment:
Add this behind a feature flag (or just remove this line for now) so that you can merge this branch to dev. That way you avoid conflicts later (even though they should be minimal).

Reply:
Done in https://github.com/HedvigInsurance/android/pull/1433